app/models/station.rb

```diff
@@ -37,6 +37,7 @@ class Station < ActiveRecord::Base
   #validates_presence_of :hw_id
   validates_numericality_of :speed_calibration
   validates_numericality_of :balance, allow_blank: true
+  validates_numericality_of :sampling_rate, allow_blank: true, max: 24.hours.to_i
```

Comment:
Could you please add some description of what sampling_rate is used for or what the purpose of it is? Is it a value that the Arduino clients are expected to read and follow to sample the wind speed and direction? It does not seem to be that but rather some value indicating the expected update interval of observations from an Arduino client. What is the effect of changing it? Is it just used to determine if a station should be set to offline state?